app/api/resources/user.py

```diff
@@ -260,11 +260,13 @@ class UserRegister(Resource):
     @users_ns.response(HTTPStatus.OK, "%s" % messages.USER_WAS_CREATED_SUCCESSFULLY)
     @users_ns.response(
         HTTPStatus.BAD_REQUEST,
-        "%s\n%s\n%s"
+        "%s\n%s\n%s\n%s"
         % (
             messages.USER_USES_A_USERNAME_THAT_ALREADY_EXISTS,
             messages.USER_USES_AN_EMAIL_ID_THAT_ALREADY_EXISTS,
+            messages.USERNAME_FIELD_IS_MISSING,
```

Comment:
Add this message  `The username field has to be longer than 4 characters and shorter than 26 characters.` instead of `USERNAME_FIELD_IS_MISSING`

For this
- Create a new message variable in `app/messages.py` file
- add this message variable here in this file.

Reply:
@satya7289, Thank you for the help. Originally, I used the already existing `USERNAME_FIELD_IS_MISSING` error message. However,I have added a `USERNAME_FIELD_IS_EMPTY` error in `app/messages.py,` It should be fine now.